geth/notification/response.go

```diff
@@ -0,0 +1,33 @@
+package notification
+
+import "github.com/NaySoftware/go-fcm"
+
+// Response is a copy of fcm.FcmResponseStatus, good to have it as default type fore notification response
+type Response struct {
+	Ok           bool
+	StatusCode   int
+	MulticastID  int64               `json:"multicast_id"`
+	Success      int                 `json:"success"`
+	Fail         int                 `json:"failure"`
+	CanonicalIDs int                 `json:"canonical_ids"`
+	Results      []map[string]string `json:"results,omitempty"`
+	MsgID        int64               `json:"message_id,omitempty"`
+	Err          string              `json:"error,omitempty"`
+	RetryAfter   string
+}
+
+// FromFCMResponseStatus turns FCM response status into generic Response object
+func FromFCMResponseStatus(f *fcm.FcmResponseStatus) *Response {
```

Comment:
I think that `FirebaseClient` should be responsible for constructing a  correct response.